src/pages/edge/adobe-journey-optimizer-decisioning/assurance-setup.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Setting up Decisioning plugins in Assurance
+description: This document shows user how to setup Assurance for Mobile SDK.
+keywords:
+- Assurance
+- Decisioning
+---
+
+# Setting up Decisioning plugins in Assurance
+
+This document shows users how to set up Assurance to be used with Decisioning Mobile SDK.
+
+## Setup
+
+1. To get started you will need the Adobe Experience Platform Assurance extension to be installed in your mobile application. Follow the instruction provided in [**Assurance Overview**](../../home/base/assurance/index.md) for setup.
+
+2. Next, navigate to the Assurance UI inside Data Collection, you will need to create and connect to an Assurance session. You can follow instructions provided [**Connecting to Assurance**](https://experienceleague.adobe.com/en/docs/experience-platform/assurance/tutorials/using-assurance) tutorial.
+
+   Once your mobile application is connected to assurance, you should see the Assurance Session Bubble on the devices connected to the session and the number of connected devices in the Assurance portal.
+
+   ![](./assets/assurance-setup/assurance-connected.png)
+
+3. If you do not see Decisioning plugins in the **Assurance plugins section**, on the left side of the page. Ensure that they are properly configured using the panel **Configure** menu, available at the bottom of the panel.
```

Comment:
Don't think we need a period between `page. Ensure`. It should be one sentence

Reply:
panel **Configure** menu seems a bit refundant. We can say using the **Configure** button available on...